app/_ui/enrolment/enrolment.component.tsx

```diff
@@ -219,8 +219,18 @@ export function EnrolmentContent({studyProtocol}: EnrolmentContentProps) {
         (res) => {
           if (res.ok) {
             sendGAEvent('event', 'study_enrolment', {status: 'joined'})
-            router.push("portal")
-            window.location.reload()
+            res.json().then(
+              (data) => {
+                const verificationFlow = data.continue_with[0].flow.id
+                if (verificationFlow) {
+                  router.push(`/${studyProtocol.studyId}/verification?flow=${verificationFlow}`)
```

Comment:
Should this go to `/auth/verification`?

Reply:
I thought being able to use either /auth/verification or /studyId/verification would be useful because the /studyId/ will use the study's theme. Only study participants should be redirected there (e.g. in this case because they just enrolled). The actual page is just a wrapper around the verification component.

Reply:
Oh ok I see, sounds good. 👍